spring-core/src/main/java/com/baeldung/factorybean/InitializationToolFactory.java

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.factorybean;
+
+import org.springframework.beans.factory.FactoryBean;
+import org.springframework.beans.factory.InitializingBean;
+
+public class InitializationToolFactory implements FactoryBean<Tool>, InitializingBean {
+    private int factoryId;// standard setters and getters
```

Comment:
Again,
Please remove this comments //standard setters and getters.

Such comment should only be present in code samples in the article, but not in the source code.